src/os/bluestore/BlueStore.cc

```diff
@@ -12415,12 +12415,14 @@ void BlueStore::_flush_cache()
 // We use a best-effort policy instead, e.g.,
 // we don't care if there are still some pinned onodes/data in the cache
 // after this command is completed.
-void BlueStore::flush_cache()
+int BlueStore::flush_cache(ostream *os)
```

Comment:
Looks like the parameter "os" is just ignored. Consider just not taking it, or perhaps using std::optional<>. (Bare pointer parameters make me sad.)

Reply:
The signature of the virtual method changed.

Reply:
@mogeb please add `override` tot his method.

Reply:
+1

Reply:
There already is the `override` keyword in the header files, this is the .cc file.

Reply:
Oh whoops. Sorry for the noise.